stream-chat-android-ui-components/src/main/res/values-hi/strings_message_input.xml

```diff
@@ -24,4 +24,4 @@
   <string name="stream_ui_message_input_send_as_direct_message">"सीधे मैसेज के रूप में भी भेजें"</string>
   <string name="stream_ui_message_input_send_to_channel">"चैनल पर भी भेजें"</string>
   <string name="stream_ui_message_input_slow_mode_hint">"स्लो मोड ऑन"</string>
-</resources>
\ No newline at end of file
+</resources>
```

Comment:
If the newline is not added from autoformat, we may avoid adding these newlines because Localazy doesn't like it.